packages/cli/generators/copyright/license.js

```diff
@@ -40,8 +40,6 @@ This project is licensed under the ${license.name}, full text below.
 
 --------
 
-${license.name}
-
```

Comment:
I find this slightly suspicious. Take a look at https://github.com/strongloop/loopback-next/blob/master/LICENSE, it starts with the following header:

```
Copyright (c) IBM Corp. 2017,2019.
Node module: loopback-next
This project is licensed under the MIT License, full text below.

--------

MIT license

(...)
```

I always thought the MIT license text is supposed to start with the license name?

Reply:
It is already integrated in the new version of this dependency